spec/models/dialog_field_serializer_spec.rb

```diff
@@ -147,7 +150,8 @@
                      :category_id          => "123",
                      :category_name        => "best category ever",
                      :category_description => "best category ever"
-                   }
+                   },
+                   "default_value"           => default_values.split(",")
```

Comment:
Minor change, but I think this should be hard-coded to `%w(one two)`. There's really no way the `.split` method is going to change, but I feel like matching the test expected output with what the code is actually doing doesn't really set the expectation correctly.